create-table.md

```diff
@@ -4,87 +4,89 @@ toc: false
+`index_def` | An optional, comma-separated list of [index definitions](data-definition.html#indexes). For each index, the column(s) to index must be specified; optionally, a name can be specified. Index names must be unique within the table but can have the same name as columns or constraints. See the [Create Indexes with a Table](#create-indexes-with-a-table) example below.<br><br>The [`CREATE INDEX`](create-index.html) statement can be used to create an index separate from table creation.  
+`table_constraint` | An optional, comma-separated list of [table-level constraints](data-definition.html#table-level-constraints). Constraint names must be unique within the table but can have the same name as columns or indexes.
 
 ## Examples
 
-If the table does **not** have a `PRIMARY KEY` constraint defined, a mandatory column called `rowid` of type `INT` will be added and automatically populated with a unique id that will be used as the primary key. The column will be added even if the table has an `UNIQUE` constraint.
+### Create a Table With a Primary Key
+
+In this example, we create a table with three columns. One column is given the [`PRIMARY KEY`](data-definition.html#primary-key) constraint, another is given the [`UNIQUE`](data-definition.html#unique) constraint, and the third has no constraints. Indexes are automatically created for columns with the `PRIMARY KEY` and `UNIQUE` constraints.
+
+~~~ 
+CREATE TABLE logoff (user_id INT PRIMARY KEY, user_email STRING UNIQUE, logoff_date DATE);
+CREATE TABLE
+
+SHOW COLUMNS FROM logoff;
++-------------+--------+-------+---------+
+|    Field    |  Type  | Null  | Default |
++-------------+--------+-------+---------+
+| user_id     | INT    | false | NULL    |
+| user_email  | STRING | true  | NULL    |
+| logoff_date | DATE   | true  | NULL    |
++-------------+--------+-------+---------+
+
+SHOW INDEX FROM logoff;
++--------+-----------------------+--------+-----+------------+-----------+---------+
+| Table  |         Name          | Unique | Seq |   Column   | Direction | Storing |
++--------+-----------------------+--------+-----+------------+-----------+---------+
+| logoff | primary               | true   |   1 | user_id    | ASC       | false   |
+| logoff | logoff_user_email_key | true   |   1 | user_email | ASC       | false   |
++--------+-----------------------+--------+-----+------------+-----------+---------+
+~~~
+
+### Create a Table Without a Primary Key
+
+In this example, we create a table with two columns, neither of which are given the the [`PRIMARY KEY`](data-definition.html#primary-key) constraint. As a result, a column called `rowid` of the type `INT` is added automatically as the primary key, with the `unique_row(id)` function used to ensure that new rows always default to unique `rowid` values. Also, an index is automatically created for `rowid`, since it is the primary key. 
```

Comment:
I might add something to basically saying that a PRIMARY KEY is _required_ because of how we store rows, which is why we add the extra field (which should also hopefully suggest to the reader that they should really be specifying their own).